### PR TITLE
Updated dependcies, examples, builder API usage

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

**Comment by stoiet** [comment]
Nice!

File: index.js [hunk 1]
```diff
@@ -18,11 +18,14 @@ function DevBuilder(options) {
...
 
-  this.builder.invalidate(filename);
-
-  this.logInfo('Invalidated', chalk.blue(filename), 'from cache');
+  var invalidated = this.builder.invalidate(filename);
+  if (invalidated.length === 0) {
+    return false;
+  } else {
+    return true;
+  }
```
Comment (jackfranklin) [inline index.js]: can rewrite this as: `return invalidated.length > 0`

**Comment by jackfranklin** [comment]
@OrKoN amazing, thank you so much!

One small comment, and also can you add yourself to the contributors part of the package.json so you get credit? Then I'll merge and push a new version.

**Comment by OrKoN** [comment]
@jackfranklin Thanks! updated

**Comment by jackfranklin** [comment]
Published as 0.3.1

**Pull request merged**